extensions/utility.py

```
@@ -765,8 +775,8 @@ async def schedulemessage(  # type: ignore[no-untyped-def]
             client=ctx.client,
         )
 
-        attachments: list[Any] = []  # [a for a in [attachment1, attachment2, attachment3, attachment4, attachment5,
-        #                          attachment6, attachment7, attachment8, attachment9, attachment10] if a is not None]
+        attachments: list[discord.Attachment] = [a for a in [attachment1, attachment2, attachment3, attachment4, attachment5,
+                                        attachment6, attachment7, attachment8, attachment9, attachment10] if a is not None]
```

Flake8 (7.3.0)

[error] continuation line under-indented for visual indent

(E128)